geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/TemporalTiledRasterLayer.scala

```diff
@@ -123,6 +123,7 @@ class TemporalTiledRasterLayer(
 
   def tileToLayout(
     layoutDefinition: LayoutDefinition,
+    zoom: Option[Int],
     resampleMethod: ResampleMethod
   ): TiledRasterLayer[SpaceTimeKey] = {
```

Comment:
Is this call to `.tile_to_layout` uncovered now ? https://github.com/locationtech-labs/geopyspark/blob/master/geopyspark/geotrellis/layer.py#L997